UniversityofHelsinki/Styleguide
# Accessability improvements

## Description
- Accessability improvements. See pdf below for more information.
- Fixed gulp serve command to not go into infinite loop. 

## How to test
1. checkout branch and `gulp serve`
2. confirm colors are changed and that other defined changes are correct (See related pdf below)

[hy-style-guide_improvement-proposals-2.pdf](https://github.com/UniversityofHelsinki/Styleguide/files/2754995/hy-style-guide_improvement-proposals-2.pdf)
